css-canvas-medical/index.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+
+    <div class="row" id="nav-bar">
+      <div id="canvas-nav-bar" class="column-quarter">
+        <img src="images/logo-blue.svg" alt="image-blue">
+      </div>
+
+      <div class="column-quarter" id="nav-white-space"></div>
+
+      <div class="column-quarter" id="nav-bar-two-buttons">
+        <input class="product-input" type="submit" name="product" value="Product">
+        <input class="implementation" type="submit" name="implementation" value="Implementation">
+      </div>
+      <div class="column-quarter" id="nav-bar-three-buttons">
+        <input class="about-us" type="submit" name="about-us" value="About Us">
+        <input class="blog" type="submit" name="blog" value="Blog">
+        <input class="get-started" type="submit" name="learn-more-submit" value="Contact Us">
+      </div>
```

Comment:
These can be links (`a`) instead of inputs. Also, your navbar can be just two columns. Look into flexbox to get even spacing between things.